.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,25 +21,25 @@ Steps to reproduce the behavior
 A clear and concise description of what you expected to happen.
 
 **System information (please complete the following information):**
- - OS and version: (e.g. Fedora 29 or `$ cat /etc/system-release`)
- - `# uname -a`
- - `# rpm -qa "*leapp*"` (or shorthashes of commits in case of manual installation):
+
+- OS and version: (e.g. Fedora 29 or `$ cat /etc/system-release`)
+- `# uname -a`
+- `# rpm -qa "*leapp*"` (or shorthashes of commits in case of manual installation):
+
 ```
 # rpm -qa "leapp*"
 leapp-0.4.0-1.201812151505Z.87a7b74.master.el7_5.noarch
 leapp-repository-data-0.4.0-1.201812131256Z.f25d14e.master.el7_5.noarch
 leapp-repository-0.4.0-1.201812131256Z.f25d14e.master.el7_5.noarch
 ```
- - attach (or provide link to) log files if applicable (optional - may contain confidential information):
-   - **_/var/log/upgrade.log_**
-   - */tmp/leapp-report.txt*
-   - `# tar -czf dnf-debug.tgz /tmp/download-debugdata`
-   - */var/lib/leapp/leapp.db*
-   - *journalctl*
- - anything else you would like to provide (e.g. storage info):
+
+- attach (or provide link to) log files if applicable (optional - may contain confidential information):
+  - `# tar -czf dnf-debug.tgz /var/log/leapp/`
```

Comment:
This will pack also the archive tarballs. I'm not sure if we want that. You can tweak the tarballing command (for example by employing `find`) or keep it simple & stupid:
```
- /var/log/leapp/dnf-debugdata/
- /var/log/leapp/leapp-upgrade.log
- /var/log/leapp/leapp-report.txt
- /var/log/leapp/dnf-plugin-data.txt
- /var/lib/leapp/leapp.db
- journalctl
```

Reply:
`tar -czf dnf-debug.tgz /var/log/leapp/*.{txt,log} /var/log/leapp/dnf-debugdata/ /var/lib/leapp/leapp.db`